### PR TITLE
ci: verify that code generation works and results match

### PR DESCRIPTION
**Pull request by askervin**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -51,7 +51,14 @@ jobs:
           echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
 
       - run: |
+          make install-protoc-dependencies install-protoc install-ttrpc-plugin
```
Comment (klihub) [inline .github/workflows/ci.yml:54]: Might need to do it, like
```
   - run: |
       make install-protoc-dependencies install-ttrpc-plugin
       sudo make install-protoc
       ... then the rest
```

Comment (askervin) [reply .github/workflows/ci.yml]: Thanks, fixed!

**Review by klihub — APPROVED** [review]
Thanks you, Antti ! LGTM.

File: Makefile [hunk 2]
```diff
@@ -166,6 +166,13 @@ vet:
 golangci-lint:
 	$(Q)$(GOLANG_CILINT) run
 
+validate-repo:
```
Comment (mikebrow) [inline Makefile:169]: nit.. validate-repo-changes-are-staged  
```# used after rebuilding protos to confirm related changes are in the commit```

**Review by mikebrow — APPROVED** [review]
LGTM
nit when looking at the makefile the new target isn't as obvious as it is when looking at the protoc force regen test.. see proposed de minimis edit

**Review by klihub — APPROVED** [review]
LGTM

**Review by mikebrow — APPROVED** [review]
LGTM Thanks!

**Pull request merged**